kind local registry setup update on Readme.md (#453)
knative-extensions/kn-plugin-quickstart: README.md

``` bash
kn quickstart kind
```

> For [KinD](https://kind.sigs.k8s.io/), a local registry will no longer be created by default. Users will need to pass the `--registry` flag if they wish to create a local registry:

Reviewer: let's rephrase this a bit so it describes what can be done, not what no longer can be done

```suggestion
Kind can be configured with a [local container image registry](https://kind.sigs.k8s.io/docs/user/local-registry/) by passing the `--registry` flag:
```

```bash
kn quickstart kind --registry
```

> 

Reviewer: In light of the changes made in #468 , it would also be good to add a note about tag resolution:

```suggestion

Note: we automatically configure tag resolution for the local registry when this flag is passed.
```
